k8s-tests_istio.io_release-1.6_postsubmit fails 100% of the time

https://prow.istio.io/job-history/gs/istio-prow/logs/k8s-tests_istio.io_release-1.6_postsubmit

The issue happens because we have:
```
test.kube.postsubmit: test.kube.presubmit
```
and then

```
test.kube.%: init | $(JUNIT_REPORT)
	$(eval INTEGRATION_TEST_FLAGS += --istio.test.tag=$(TAG))
	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/$(subst .,/,$*)/... -timeout 30m \
	--istio.test.select -postsubmit,-flaky \
	--istio.test.env kube \
	${_INTEGRATION_TEST_FLAGS} \
	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT)
```
and so we see:
```
PATH=/usr/local/go/bin:/gobin:/usr/local/google-cloud-sdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/prow/go/src/istio.io/istio.io/out/linux_amd64 go test -p 1 -v ./tests/postsubmit/... -timeout 30m \
--istio.test.select -postsubmit,-flaky \
--istio.test.env kube \
--istio.test.tag=1.6-alpha.1e8c62baad31c829c5993235c429248f36a85478 --istio.test.tag=1.6-alpha.1e8c62baad31c829c5993235c429248f36a85478 --istio.test.ci --istio.test.pullpolicy=IfNotPresent --istio.test.kube.minikube --istio.test.work_dir=/logs/artifacts --istio.test.hub=gcr.io/istio-testing --istio.test.kube.config=~/.kube/config \
2>&1 | tee >(/usr/bin/go-junit-report > /logs/artifacts/junit.xml)
go: warning: "./tests/postsubmit/..." matched no packages
no packages to test
tests/tests.mk:44: recipe for target 'test.kube.postsubmit' failed
```

--- a/tests/tests.mk
+++ b/tests/tests.mk
@@ -40,7 +40,7 @@ test.kube.presubmit: init | $(JUNIT_REPORT)
 
 test.kube.postsubmit: test.kube.presubmit
 
-test.kube.%: init | $(JUNIT_REPORT)
+test.kube.directory.%: init | $(JUNIT_REPORT)
 	$(eval INTEGRATION_TEST_FLAGS += --istio.test.tag=$(TAG))
 	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/$(subst .,/,$*)/... -timeout 30m \
 	--istio.test.select -postsubmit,-flaky \

--- a/tests/tests.mk
+++ b/tests/tests.mk
@@ -39,12 +39,3 @@ test.kube.presubmit: init | $(JUNIT_REPORT)
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
 test.kube.postsubmit: test.kube.presubmit
-
-test.kube.directory.%: init | $(JUNIT_REPORT)
-	$(eval INTEGRATION_TEST_FLAGS += --istio.test.tag=$(TAG))
-	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/$(subst .,/,$*)/... -timeout 30m \
-	--istio.test.select -postsubmit,-flaky \
-	--istio.test.env kube \
-	${_INTEGRATION_TEST_FLAGS} \
-	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
-

